src/pages/create-test/FieldsComponents/CheckBoxField.tsx

```diff
@@ -0,0 +1,27 @@
+import { FormControlLabel, FormGroup, Radio, Stack } from '@mui/material';
+import React from 'react';
+import { BasketIcon } from '../../../assets/icons/BasketIcon';
+
+export function CheckBoxField() {
+  return (
+    <FormGroup>
+      <Stack direction='row' alignItems='center'>
+        <FormControlLabel
+          control={
+            <Radio
+              sx={{
+                '&, &.Mui-checked': {
+                  color: 'secondary.main',
+                },
+              }}
+            />
+          }
+          label=''
```

Comment:
maybe you need label from props?

Reply:
Delete them, couse this is unused component.